concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1323,7 +1323,7 @@ jobs:
                   mkdir -p /var/vcap/jobs/acceptance-tests/bin/ /var/vcap/packages/acceptance-tests/src/github.com/cloudfoundry
                   ln -snf $(pwd)/test-config/config.json /var/vcap/jobs/acceptance-tests/bin/config.json
                   ln -snf $(pwd)/test-config/run /var/vcap/jobs/acceptance-tests/bin/run
-                  ln -snf /usr/local/go /var/vcap/packages/golang1.4
+                  ln -snf /usr/local/go /var/vcap/packages/golang1.6
                   sed -i 's/bits=@"%s"/bits=@%s/' cf-release/src/github.com/cloudfoundry/cf-acceptance-tests/helpers/v3_helpers/v3.go
```

Comment:
The commit message should probably contain that this is for acceptance-tests and why we need to bump the version. This commit could be probably combined with the commit that's bumping the go version for smoke tests, as I think the reasons for bumping the version could be the same (...as this is same shared dependency for both tests).